src/play/model/Game.py

```diff
@@ -175,60 +167,10 @@ def play(self, move: Move, player: {'w', 'b'}, testing=False):
             self.board_history.add(test_board.to_number())
             self.play_history.append(player + ':' + str(move))
 
-    def _str2index(self, loc: str) -> Tuple[int, int]:
-        """Convert the sgf location format to a usable matrix index
-
-        Examples
-        --------i
-        >>> g = Game(); g._str2index('ef')
-        (5, 4)
-        """
-        col = self._chr2ord(loc[0])
-        row = self._chr2ord(loc[1])
-        return (row, col)
-
-    def _index2str(self, index: Tuple[int, int]) -> str:
-        """Convert the sgf location format to a usable matrix index
-
-        Examples
-        --------
-        >>> g = Game(); g._str2index('ef')
-        (5, 4)
-        """
-        col = self._ord2chr(index[1])
-        row = self._ord2chr(index[0])
-        return col+row
-
     def __str__(self):
-        """Game representation = Board representation!"""
+        """Game representation = Board representation"""
```

Comment:
🤔  For now at least? We might want to add more things to the game representation like the number of captured stones and players names etc.?

Reply:
Yes, that's what I also thought. We could talk about what we actually want to see, and then add those. The method will still call `str(board)` at some point I guess :)